sys-devel/gettext/gettext-0.19.7.recipe

```diff
@@ -97,7 +107,6 @@ INSTALL()
 	make install AUTOCONF=: AUTOMAKE=: AUTOHEADER=: ACLOCAL=:
 
 	rm $libDir/charset.alias
-	rm $libDir/lib*.la
```

Comment:
We can keep *.la when they are needed. The only way to know that was to remove them first :=)
BTW when keeping *.la, make sure the paths they might contain are correct (no packages references for instance).